shotover-proxy/tests/test-configs/cassandra-passthrough-parse-request/topology.yaml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Could we combine these `topology.yaml` files into the same folder. I don't think they're different enough to warrant separate ones.

Reply:
I think the naming makes a lot more sense when we can give each its own folder and call the file topology.yaml
Otherwise you cant tell from the filename if its a toplogy file or docker-compose file :cry: